massa-ledger-worker/src/ledger_db.rs

```diff
@@ -493,6 +501,20 @@ impl LedgerDB {
         batch.ledger_hash ^= Hash::compute_from(&slot_bytes);
     }
 
+    pub fn get_slot(&self) -> Result<Slot, ModelsError> {
+        let handle = self.db.cf_handle(METADATA_CF).expect(CF_ERROR);
+
+        self.db.get_pinned_cf(handle, SLOT_KEY).expect(CRUD_ERROR);
+
+        let Ok(Some(slot_bytes)) = self.db.get_pinned_cf(handle, SLOT_KEY) else {
```

Comment:
same db call made twice?

Reply:
Good catch :)
Fixed in 5596b96fea113585dfab2b4ee7dc5165ac044db4

Reply:
nice.